expense-authorization/src/UI/angular-eteam/src/environments/environment.prod.ts

```diff
@@ -1,4 +1,4 @@
 export const environment = {
   production: true,
-  apiEndpoint: ''
+  apiEndpoint: 'http://eteams-api/api'
```

Comment:
Not sure how this is going work, the generated code from swagger already has `/api` in the path, see https://github.com/bcgov/embc-eaf/blob/1db82e0492d2492e046f7fcff720c2fd23ae1141/expense-authorization/src/UI/angular-eteam/src/app/api/generated/api/expenseAuthorization.service.ts#L232

Perhaps it should be just `http://eteams-api`